hare/algorithm.go

```diff
@@ -739,6 +730,11 @@ func (proc *consensusProcess) processCommitMsg(ctx context.Context, msg *Msg) {
 }
 
 func (proc *consensusProcess) processNotifyMsg(ctx context.Context, msg *Msg) {
+	if proc.notifyTracker == nil {
```

Comment:
does it make sense to assign one whichever (beginNotifyRound or processNotifyMsg) happens first?